src/Components/Artwork/Details.tsx

```diff
@@ -321,6 +348,14 @@ export const DetailsFragmentContainer = createFragmentContainer(Details, {
       date
       sale_message: saleMessage
       cultural_maker: culturalMaker
+      artist {
+        targetSupply {
+          isP1
+        }
+      }
+      marketPriceInsights {
+        demandRank
+      }
```

Comment:
**question:** I'm concerned that this might cause the artwork list across Force will be a bit slower since with every request to MP (and gravity) there will be another one to Vortex to get the market price insights, is there a way to have a conditional way to query only if `showHighDemandInfo` is true? cc @olerichter00

Reply:
Hmmmm. Good point 👍 . Curious what the others think too.